scriptmodules/ports/rigelengine.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+# This file is part of RetroPie-Extra, a supplement to RetroPie.
+# For more information, please visit:
+#
+# https://github.com/RetroPie/RetroPie-Setup
+# https://github.com/Exarkuniv/RetroPie-Extra
+#
+# See the LICENSE file distributed with this source and at
+# https://raw.githubusercontent.com/Exarkuniv/RetroPie-Extra/master/LICENSE
+#
+
+rp_module_id="rigelengine"
+rp_module_desc="RigelEngine - Duke Nukem 2 source port"
+rp_module_help="Copy your game to roms/ports/duke2 folder"
+rp_module_licence="GPL2 https://github.com/lethal-guitar/RigelEngine/blob/master/LICENSE.md"
+rp_module_repo="git https://github.com/lethal-guitar/RigelEngine.git v0.9.1"
+rp_module_section="exp"
+rp_module_flags="noinstclean"
+
+
+function depends_rigelengine() {
+    getDepends cmake libsdl2-dev libsdl2-mixer-dev
+}
+
+
+function sources_rigelengine() {
+    gitPullOrClone
+}
+
+function build_rigelengine() {
+    mkdir $md_build/build
+    cd $md_build/build
+    cmake .. -DUSE_GL_ES=ON -DCMAKE_BUILD_TYPE=Release -DWARNINGS_AS_ERRORS=OFF
+
+    make
+
+    md_ret_require="$md_build/build/src/RigelEngine"
+}
+
+function install_rigelengine() {
+    md_ret_files=(
+        build/src/RigelEngine
+    )
+}
+
+function game_data_rigelengine() {
+    if [[ ! -f "$romdir/ports/duke2/NUKEM2.EXE" ]]; then
+        downloadAndExtract "https://github.com/Exarkuniv/game-data/raw/main/duke2.zip" "$romdir/ports/duke2/"
+    #mv "$romdir/ports/duke2/Duke2"* "$romdir/ports/duke2/"
+    chown -R $user:$user "$romdir/ports/duke2"
```

Comment:
Wrong indent and commented out line that should be removed if not needed.

Reply:
remove the line, i get. but what do you mean by wrong indent?

Reply:
> what do you mean by wrong indent?

Since it's inside the `if` statement, `chown` should be indented four additional spaces to line up with `downloadAndExtract`.